build/Defaults/Desktop/app.config

```diff
@@ -73,6 +73,54 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral"/>
         <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0"/>
       </dependentAssembly>
+
+      <!-- Even though all of our references are to Dev15 binaries we are still pulling in the interactive window 
```

Comment:
Looks to be 2.0.0-rc3-61304-01: https://dotnet.myget.org/gallery/interactive-window